.github/workflows/gh-pages.yml

```diff
@@ -0,0 +1,36 @@
+name: GitHub Pages
+
+on:
+    push:
+        branches:
+            - "master"
+        paths:
+            - ".github/workflows/update-docs.ya?ml"
+            - "mkdocs.ya?ml"
+            - "poetry.lock"
+            - "pyproject.toml"
+            - "docs/**"
+
+jobs:
+    publish:
+        runs-on: ubuntu-latest
+        permissions:
+            contents: write
+
+        steps:
+            - name: Checkout repository
+              uses: actions/checkout@v4
+
+            - name: Install Python
+              uses: actions/setup-python@v5
+              with:
+                  python-version: ${{ env.PYTHON_VERSION }}
```

Comment:
Where is `PYTHON_VERSION` set?

Reply:
Using static `3.12` now.

Reply:
OK. But where was it defined first? :confused:

Reply:
...

Probably wherever I copy pasted it from 😅🤣